algorithms/ar-expsq/golang/exponentiation_by_square.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import "fmt"
+
+var mod int = 1000000007
```

Comment:
Codacy found an issue: [should omit type int from declaration of var mod; it will be inferred from the right-hand side](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6393195)

Reply:
Fixed. Please have a look